src/python/geoclaw/surge/storm.py

```diff
@@ -178,6 +374,67 @@ def write_imd(self, path):
         r""""""
         raise NotImplementedError("IMD format not fully implemented.")
 
+    # Useful functions
+    
+    def date2seconds(self,date): 
```

Comment:
I think these utility functions should be module level functions and not class methods.  Some of these I think also might be in the `util` module somewhere.